ansible/roles/debops.dokuwiki/defaults/main.yml

```diff
@@ -381,6 +381,37 @@ dokuwiki__protected_conf_php: |
   {% endif %}
 # ]]]
 # ]]]
+# MIME Types local configuration [[[
+# ---------------------------------
+
+# .. envvar:: dokuwiki__local_mime_types_conf [[[
+#
+# Additional allowed uploadable file extensions and mimetypes are defined here.
+# Mimetypes that should be downloadable and not be opened in the wiki should
+# be prefixed with a ``!``.
+#
+# Example:
+#
+# .. code-block:: yaml
+#
+#      # Extra allowed MIME Types
+#      txt     text/plain
+#      tex     text/plain
+#      conf    text/plain
+#      xml     text/xml
+#      csv     text/csv
+#      svg     image/svg+xml
+#      epub    application/epub+zip
```

Comment:
I think that these are reasonable choices for a default set, perhaps you could move them to the variable instead of providing them as an example?